### DISCUSSION TITLE
Is ImageGlass safe from the recent WebP vulnerability? (CVE-2023-4863)

### DISCUSSION THREAD

**Discussion by gaeulbyul** [d2phap/ImageGlass#1655] [Q&A] [answered]
Currently I'm using ImageGlass v8.9.6.9 on Windows 10.

I heard about the recent webp vulnerability (CVE-2023-4863) [affects many applications](https://stackdiary.com/critical-vulnerability-in-webp-codec-cve-2023-4863/) and I wonder if ImageGlass also affects this vulnerability. Should I worry about it?

**Comment by d2phap** [comment]
Both ImageGlass 8.9 and 9.0 beta 5 are affected. I'm preparing an update to fix it.

**Reply by d2phap** [reply, marked as answer]
ImageGlass 8.10 is released!
https://github.com/d2phap/ImageGlass/releases/tag/8.10.9.27
